packages/fiori/src/themes/NotificationListGroupItem.css

```diff
@@ -41,6 +41,10 @@
 	cursor: pointer;
 }
 
+.ui5-nli-group-header[aria-expanded="false"] {
```

Comment:
Please add new attribute (expanded/collapse for example) and use it here. If the ACC is changed the height will no longer work.